fedmsg_meta_fedora_infrastructure/compose.py

```diff
@@ -76,9 +118,19 @@ def link(self, msg, **config):
         else:
             if 'epelbeta' in msg['topic']:
                 base = "https://dl.fedoraproject.org/pub/epel/beta/7"
-            else:
+            elif 'branched' in msg['topic'] or 'rawhide' in msg['topic']:
                 base = "https://dl.fedoraproject.org/pub/" + \
                     "fedora/linux/development"
+            else:
+                branch = msg['msg'].get('branch', 'not-a-number')
```

Comment:
No - "not-a-number", because I try to cast it to an ``int`` just two lines below.